packages/ng/libraries/root/src/style/definitions/select/_select-input.scss

```diff
@@ -124,6 +124,7 @@
 				padding: 0 calc(#{_component("select.framed.side-padding")} * 3)
 					_component("select.framed.bottom-padding")
 					_component("select.framed.side-padding");
+				min-height: 1.5rem;
```

Comment:
thats all it took ?

Reply:
Can you turn that into a variable ? It might vary if people changes other variables :)